Gem/Code/Source/AutoGen/PlayerJumpPadEffectComponent.AutoComponent.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+
+<Component
+    Name="PlayerJumpPadEffectComponent"
+    Namespace="MultiplayerSample"
+    OverrideComponent="true"
+    OverrideController="true"
+    OverrideInclude="Source/Components/Multiplayer/PlayerJumpPadEffectComponent.h"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <ComponentRelation Constraint="Required" HasController="true" Name="NetworkTransformComponent" Namespace="Multiplayer" Include="Multiplayer/Components/NetworkTransformComponent.h" />
+    <ComponentRelation Constraint="Required" HasController="true" Name="NetworkCharacterComponent" Namespace="Multiplayer" Include="Multiplayer/Components/NetworkCharacterComponent.h" />
+
+    <ArchetypeProperty Type="float" Name="Gravity" ExposeToEditor="true"
```

Comment:
I believe you can query this from physx, which means if a designer makes a bunch of different levels with different gravity settings, this component will still respect those settings. Think like Prey, where they had a whole tensor space setup for gravity throughout the world volume.

Reply:
Yup, just double-checked because we do ballistic calculations for bullet drop on trace weapons.

```
        // World gravity for our current location (making the currently safe assumption that it's constant over the duration of our trace)
        AzPhysics::SceneInterface* sceneInterface = AZ::Interface<AzPhysics::SceneInterface>::Get();
        AzPhysics::SceneHandle sceneHandle = sceneInterface->GetSceneHandle(AzPhysics::DefaultPhysicsSceneName);
        const AZ::Vector3& gravity = gatherParams.m_bulletDrop ? sceneInterface->GetGravity(sceneHandle) : AZ::Vector3::CreateZero();
```